From 8c468833465d7fa0d5372807de01272414b1f4bb Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Tue, 10 Oct 2017 02:55:38 -0700
Subject: [PATCH] yacc: Remove __unused

We can't define this to __attribute__((unused)) because musl uses this
identifier as a structure field.
---
 usr.bin/yacc/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr.bin/yacc/main.c b/usr.bin/yacc/main.c
index 3d9f6add5c8..dd34a04c5bf 100644
--- a/usr.bin/yacc/main.c
+++ b/usr.bin/yacc/main.c
@@ -122,7 +122,7 @@ done(int k)
 
 
 void
-onintr(__unused int signo)
+onintr(int signo)
 {
 	sigdie = 1;
 	done(1);
-- 
2.14.2

